corehq/apps/case_migrations/views.py

```diff
@@ -81,6 +83,13 @@ def migration_restore(request, domain, case_id):
             # Formplayer will be creating these cases for the first time, so
             # include create blocks
             content.append(get_case_element(case, ('create', 'update'), V2))
+        if ADD_LIMITED_FIXTURES_TO_SMS_WORKFLOW.enabled(domain):
+            _add_limited_fixtures(content)
         payload = content.get_fileobj()
 
     return RestoreResponse(payload).get_http_response()
+
+
+def _add_limited_fixtures(content):
+    serializer = HierarchicalLocationSerializer()
+    content.extend(serializer.get_xml_nodes('commtrack:locations', None, SQLLocation.active_objects, None))
```

Comment:
Note: We can't use the usual location fixture providers used in restore, https://github.com/dimagi/commcare-hq/blob/1b5625a828f5ad1772fa721687704f0bfd8a5e9e/corehq/apps/locations/fixtures.py#L234-L239  because that needs a restore user, so assuming the right way would be to just generate the content here. Also seems in line with the way we are generating content for cases above.